### DISCUSSION TITLE
Errores con las librerías INCLUDE

### DISCUSSION THREAD

**Discussion by codingemmag** [INGCOM-UNRN-P1/cursada-2023#72] [Preguntas y respuestas] [answered]
Hola,

Estoy teniendo un problema con las librerías, donde por un lado me indica que tengo que actualizar el "includePath", y que además tengo que correr el comando "Select Intellisense Configuration" para ubicar los encabezados del sistema ( esta ultima parte está en ingles en realidad por eso es rara la traducción ).
Ya he editado el archivo "c_cpp_properties.json" con la ubicación del compilador del mingw y el include path del mismo, siendo "mingw64\include". Quizás ese sea el error, pero no lo creo.
Estoy usando el Visual Studio Code a propósito.

![Sin título](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/134022702/e95b16d6-fc72-4935-b1b3-20d8b393e568)

Saludos

**Comment by martinvilu** [comment, marked as answer]
Rarísimo, porque el vscode configura todo automaticamente.

La solución rápida a esto, que no es la más bonita. Es borrar la carpeta `.vscode` que se crea en el repositorio y empezar de nuevo con la autoconfiguración.

La captura es particularmente rara porque hay problemas para ubicar librerías de `C++` no de `C`

Subí el repositorio y compartí el enlace, así veo si no hay nada _intrínseco_ al código.
